<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserFlightInfo</class>
 <widget class="QWidget" name="UserFlightInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel {
    color: #333333; /* 深灰色文字，与对话框和文本输入框的文字颜色相同 */
    font-size: 14px; /* 与按钮、对话框和文本输入框相同的字体大小 */
    font-weight: normal; /* 默认字体粗细，可以根据需要调整 */
    margin: 5px 0; /* 上下外边距，根据需要调整 */
    background-color: transparent; /* 透明背景，不覆盖标签下的内容 */
}

QLabel:disabled {
    color: #888888; /* 禁用时文字颜色变暗，与禁用状态的文本输入框一致 */
}</string>
              </property>
              <property name="text">
               <string>起点</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="StartPoint">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    background-color: #e0f7fa; /* 浅蓝色背景 */
    border: 2px solid #4fc3f7; /* 蓝色边框 */
    border-radius: 4px; /* 圆角边框 */
    padding: 5px; /* 内边距 */
    color: #005f6a; /* 深蓝色文本 */
    selection-background-color: #4fc3f7; /* 选中时的背景颜色 */
    selection-color: #ffffff; /* 选中时的文本颜色 */
}

QLineEdit:focus {
    border-color: #00796b; /* 聚焦时边框颜色变深 */
    box-shadow: 0 0 4px rgba(0, 121, 107, 0.5); /* 聚焦时阴影效果 */
}

QLineEdit:disabled {
    background-color: #d3e5f3; /* 禁用时背景颜色变浅 */
    border-color: #a1c4dd; /* 禁用时边框颜色变浅 */
    color: #6d8cac; /* 禁用时文本颜色变暗 */
}

QLineEdit::placeholder {
    color: #6d8cac; /* 占位符文本颜色 */
    font-style: italic; /* 占位符文本斜体 */
}</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel {
    color: #333333; /* 深灰色文字，与对话框和文本输入框的文字颜色相同 */
    font-size: 14px; /* 与按钮、对话框和文本输入框相同的字体大小 */
    font-weight: normal; /* 默认字体粗细，可以根据需要调整 */
    margin: 5px 0; /* 上下外边距，根据需要调整 */
    background-color: transparent; /* 透明背景，不覆盖标签下的内容 */
}

QLabel:disabled {
    color: #888888; /* 禁用时文字颜色变暗，与禁用状态的文本输入框一致 */
}</string>
              </property>
              <property name="text">
               <string>航班编号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="FlightId">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    background-color: #e6e6fa; /* 浅紫色背景 */
    border: 2px solid #9b59b6; /* 紫色边框 */
    border-radius: 4px; /* 圆角边框 */
    padding: 5px; /* 内边距 */
    color: #4b0082; /* 深紫色文本（与浅紫色背景形成对比） */
    selection-background-color: #9b59b6; /* 选中时的背景颜色（与边框颜色相同） */
    selection-color: #ffffff; /* 选中时的文本颜色（白色） */
}

QLineEdit:focus {
    border-color: #6a0dad; /* 聚焦时边框颜色变深为深紫色 */
    box-shadow: 0 0 4px rgba(106, 13, 173, 0.5); /* 聚焦时阴影效果使用深紫色RGBA */
}

QLineEdit:disabled {
    background-color: #d8bfd8; /* 禁用时背景颜色变为更浅的紫色 */
    border-color: #b09cd9; /* 禁用时边框颜色变为更浅的紫色 */
    color: #800080; /* 禁用时文本颜色变为中紫色 */
}

QLineEdit::placeholder {
    color: #800080; /* 占位符文本颜色使用中紫色 */
    font-style: italic; /* 占位符文本斜体 */
}</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel {
    color: #333333; /* 深灰色文字，与对话框和文本输入框的文字颜色相同 */
    font-size: 14px; /* 与按钮、对话框和文本输入框相同的字体大小 */
    font-weight: normal; /* 默认字体粗细，可以根据需要调整 */
    margin: 5px 0; /* 上下外边距，根据需要调整 */
    background-color: transparent; /* 透明背景，不覆盖标签下的内容 */
}

QLabel:disabled {
    color: #888888; /* 禁用时文字颜色变暗，与禁用状态的文本输入框一致 */
}</string>
              </property>
              <property name="text">
               <string>终点</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="EndPoint">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    background-color: #ffe6e6; /* 浅红色背景 */
    border: 2px solid #ff6347; /* 鲜红色边框 */
    border-radius: 4px; /* 圆角边框 */
    padding: 5px; /* 内边距 */
    color: #8b0000; /* 暗红色文本 */
    selection-background-color: #ff4500; /* 选中时的背景颜色（橙色红） */
    selection-color: #ffffff; /* 选中时的文本颜色（白色） */
}

QLineEdit:focus {
    border-color: #dc143c; /* 聚焦时边框颜色变为更深的红色 */
    box-shadow: 0 0 4px rgba(220, 20, 60, 0.5); /* 聚焦时阴影效果 */
}

QLineEdit:disabled {
    background-color: #f5dcdc; /* 禁用时背景颜色变浅 */
    border-color: #e9967a; /* 禁用时边框颜色变浅 */
    color: #a52a2a; /* 禁用时文本颜色变暗 */
}

QLineEdit::placeholder {
    color: #a52a2a; /* 占位符文本颜色（暗红色） */
    font-style: italic; /* 占位符文本斜体 */
}</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel {
    color: #333333; /* 深灰色文字，与对话框和文本输入框的文字颜色相同 */
    font-size: 14px; /* 与按钮、对话框和文本输入框相同的字体大小 */
    font-weight: normal; /* 默认字体粗细，可以根据需要调整 */
    margin: 5px 0; /* 上下外边距，根据需要调整 */
    background-color: transparent; /* 透明背景，不覆盖标签下的内容 */
}

QLabel:disabled {
    color: #888888; /* 禁用时文字颜色变暗，与禁用状态的文本输入框一致 */
}</string>
              </property>
              <property name="text">
               <string>预计起飞时间</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="StartTime">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="FlyTime">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    background-color: #e0ffe0; /* 浅绿色背景 */
    border: 2px solid #32cd32; /* 绿色边框 */
    border-radius: 4px; /* 圆角边框 */
    padding: 5px; /* 内边距 */
    color: #000000; /* 黑色文本（与浅绿色背景形成对比） */
    selection-background-color: #90ee90; /* 选中时的背景颜色（比边框稍浅的绿色） */
    selection-color: #000000; /* 选中时的文本颜色（黑色） */
}

QLineEdit:focus {
    border-color: #006400; /* 聚焦时边框颜色变深为深绿色 */
    box-shadow: 0 0 4px rgba(0, 100, 0, 0.5); /* 聚焦时阴影效果使用深绿色RGBA */
}

QLineEdit:disabled {
    background-color: #d3f5d3; /* 禁用时背景颜色变为更浅的绿色 */
    border-color: #98fb98; /* 禁用时边框颜色变为与浅绿色相协调的浅色 */
    color: #556b2f; /* 禁用时文本颜色变为深绿色（与浅绿色相协调） */
}

QLineEdit::placeholder {
    color: #556b2f; /* 占位符文本颜色使用深绿色（与浅绿色相协调） */
    font-style: italic; /* 占位符文本斜体 */
}</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="styleSheet">
               <string notr="true">QLabel {
    color: #333333; /* 深灰色文字，与对话框和文本输入框的文字颜色相同 */
    font-size: 14px; /* 与按钮、对话框和文本输入框相同的字体大小 */
    font-weight: normal; /* 默认字体粗细，可以根据需要调整 */
    margin: 5px 0; /* 上下外边距，根据需要调整 */
    background-color: transparent; /* 透明背景，不覆盖标签下的内容 */
}

QLabel:disabled {
    color: #888888; /* 禁用时文字颜色变暗，与禁用状态的文本输入框一致 */
}</string>
              </property>
              <property name="text">
               <string>小时</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel {
    color: #333333; /* 深灰色文字，与对话框和文本输入框的文字颜色相同 */
    font-size: 14px; /* 与按钮、对话框和文本输入框相同的字体大小 */
    font-weight: normal; /* 默认字体粗细，可以根据需要调整 */
    margin: 5px 0; /* 上下外边距，根据需要调整 */
    background-color: transparent; /* 透明背景，不覆盖标签下的内容 */
}

QLabel:disabled {
    color: #888888; /* 禁用时文字颜色变暗，与禁用状态的文本输入框一致 */
}</string>
              </property>
              <property name="text">
               <string>预计降落时间</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="EndTime">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="currentSection">
               <enum>QDateTimeEdit::Section::YearSection</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="widget_7" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QWidget" name="widget_8" native="true">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>60</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel {
    color: #333333; /* 深灰色文字，与对话框和文本输入框的文字颜色相同 */
    font-size: 14px; /* 与按钮、对话框和文本输入框相同的字体大小 */
    font-weight: normal; /* 默认字体粗细，可以根据需要调整 */
    margin: 5px 0; /* 上下外边距，根据需要调整 */
    background-color: transparent; /* 透明背景，不覆盖标签下的内容 */
}

QLabel:disabled {
    color: #888888; /* 禁用时文字颜色变暗，与禁用状态的文本输入框一致 */
}</string>
                 </property>
                 <property name="text">
                  <string>票价</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="PriceLine">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit {
    background-color: #fffacd; /* 浅金色背景 */
    border: 2px solid #ffd700; /* 金色边框 */
    border-radius: 4px; /* 圆角边框 */
    padding: 5px; /* 内边距 */
    color: #000000; /* 黑色文本（与金色背景形成对比） */
    selection-background-color: #ffd700; /* 选中时的背景颜色（与边框颜色相同） */
    selection-color: #000000; /* 选中时的文本颜色（黑色） */
}

QLineEdit:focus {
    border-color: #b8860b; /* 聚焦时边框颜色变深为深金色 */
    box-shadow: 0 0 4px rgba(184, 134, 11, 0.5); /* 聚焦时阴影效果使用深金色RGBA */
}

QLineEdit:disabled {
    background-color: #eee8aa; /* 禁用时背景颜色变为更浅的金色 */
    border-color: #d2b48c; /* 禁用时边框颜色变为更浅的金色 */
    color: #8b4513; /* 禁用时文本颜色变为棕色（与金色相协调） */
}

QLineEdit::placeholder {
    color: #8b4513; /* 占位符文本颜色使用棕色（与金色相协调） */
    font-style: italic; /* 占位符文本斜体 */
}</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="styleSheet">
               <string notr="true">QLabel {
    color: #333333; /* 深灰色文字，与对话框和文本输入框的文字颜色相同 */
    font-size: 14px; /* 与按钮、对话框和文本输入框相同的字体大小 */
    font-weight: normal; /* 默认字体粗细，可以根据需要调整 */
    margin: 5px 0; /* 上下外边距，根据需要调整 */
    background-color: transparent; /* 透明背景，不覆盖标签下的内容 */
}

QLabel:disabled {
    color: #888888; /* 禁用时文字颜色变暗，与禁用状态的文本输入框一致 */
}</string>
              </property>
              <property name="text">
               <string>剩余票数</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="RemainSeatsNum">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    background-color: #fff5e6; /* 浅橙色背景 */
    border: 2px solid #ffa500; /* 橙色边框 */
    border-radius: 4px; /* 圆角边框 */
    padding: 5px; /* 内边距 */
    color: #000000; /* 黑色文本（与浅橙色背景形成对比） */
    selection-background-color: #ffa07a; /* 选中时的背景颜色（比边框稍浅的橙色） */
    selection-color: #000000; /* 选中时的文本颜色（黑色） */
}

QLineEdit:focus {
    border-color: #ff4500; /* 聚焦时边框颜色变深为深橙色 */
    box-shadow: 0 0 4px rgba(255, 69, 0, 0.5); /* 聚焦时阴影效果使用深橙色RGBA */
}

QLineEdit:disabled {
    background-color: #f0e68c; /* 禁用时背景颜色变为更浅的橙色 */
    border-color: #d2b48c; /* 禁用时边框颜色变为与浅橙色相协调的浅色 */
    color: #8B4513; /* 禁用时文本颜色变为棕色（与橙色相协调） */
}

QLineEdit::placeholder {
    color: #8B4513; /* 占位符文本颜色使用棕色（与橙色相协调） */
    font-style: italic; /* 占位符文本斜体 */
}</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BuyTicketBtn">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #4CAF50; /* 绿色背景 */
    border: 2px solid #008000; /* 深绿色边框 */
    border-radius: 8px; /* 圆角边框 */
    color: #ffffff; /* 白色文本 */
    padding: 10px 20px; /* 内边距 */
    font-size: 16px; /* 字体大小 */
    min-width: 100px; /* 最小宽度 */
    transition: background-color 0.3s, border-color 0.3s; /* 平滑过渡效果 */
}

QPushButton:hover {
    background-color: #6eb46e; /* 鼠标悬停时背景颜色变为稍浅的绿色 */
    border-color: #32cd32; /* 鼠标悬停时边框颜色变为更亮的绿色 */
}

QPushButton:pressed {
    background-color: #388e3c; /* 按下时背景颜色变为深绿色 */
    border-color: #006400; /* 按下时边框颜色变为更深的绿色 */
}

QPushButton:disabled {
    background-color: #90ee90; /* 禁用时背景颜色变为浅绿色 */
    border-color: #6b8e23; /* 禁用时边框颜色变为橄榄绿 */
    color: #888888; /* 禁用时文本颜色变为灰色 */
}</string>
           </property>
           <property name="text">
            <string>购票</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
